vita3k/gui/src/themes.cpp

```diff
@@ -490,8 +496,16 @@ void draw_themes_selection(GuiState &gui, HostState &host) {
         ImGui::PopStyleVar();
         ImGui::Separator();
     } else if (menu == "theme") {
+        // Theme List
         if (selected.empty()) {
             title = "Theme";
+
+            // Set Scroll Pos
+            if (set_scroll_pos) {
+                ImGui::SetScrollY(scroll_pos);
+                set_scroll_pos = false;
+            }
```

Comment:
Strange to me why you cant just do `ImGui::SetScrollY(scroll_pos);` on line 790.

Reply:
is outside begin_child, you can't, if you set this here, you change position scroll inside begin :D